### ISSUE TITLE
When going from "locked+dimmed" to "suspend", screen is undimmed first

### ISSUE THREAD

**Issue by ydirson** [radekp/qtmoko#59]
It's a bit surprising (I wonder what's going on every time), wastes a bit of energy, and may be more than annoying when someone wants to get unnoticed :)

**Comment by radekp** [comment]
I guess this is kernel related - you can try echo mem > /sys/power/state if you see the same bahavior - i gues you will.

**Comment by ydirson** [comment]
No, I don't - there is a small "flash to white", but it is a different issue : here the backlight is switched on, for 0.5s or so.

During my hunt for the "delay on suspend" issue, I noticed a good candidate for that one - hard to find it again now though, I'll surely find it again sooner or later.
